madisonrandle/whats-cookin
# Mvp

### What does this PR do?
- Create reset.css file and add pre-written code to reset browsers styling defaults
- Style DOM to follow our wire frame
  - **Note:** Once we start working on functionality I will remove the hard coded cards from our html file 🤓
- Add .DS_Store to .gitignore file
##### Less Important Changes
- Update README.md with
  - Both of our GitHub links
  - An **Abstract** section describing the project
  - An **Instructions** section to later describe how to user the app
  - A **Setup** section with instructions for setting up and accessing the app
  - Add markdown skeletons to insert _screenshots_ and _gifs_ of the app once we're done with the project
  - A **Next Steps** section (that may be deleted later) where anything we want to complete after the project deadline
- Update the max line length in .eslintrc file

### Where should the reviewer start?
- review the index.html file and class names
- review styles.css file and comments
##### Less Important Changes
- .eslintrc file on line 21

### How should this be manually tested?
- These changes mostly updated the DOM to style our MVP. To test / see the changes, run `open index.html` in the terminal and open app in the browser 

### Potential changes?
- Add some styling bells & whistles if we have time after finishing functionality for the project 

### Screenshot
![home page mvp](https://i.postimg.cc/15TbnWp0/whats-cookin-home-page-mvp.png)

![my recipes mvp](https://i.postimg.cc/7Z2dggVF/whats-cookin-my-recipes-page-mvp.png)
